HW5.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+
+def read_gff(file_path):
+    gff = pd.read_csv(file_path, sep='\t', comment='t',
+                     names=['seqid', 'resourse', 'type', 'start', 'end', 'score', 'strand', 'frame', 'attribute'])
+    gff.drop(index = gff[gff['seqid'].str.contains('#')].index, inplace=True)
+    gff.reset_index(inplace=True)
+    gff['attribute'] = gff['attribute'].astype(str).apply(lambda x: x.split('_')[0].split('=')[1])
+    return gff
```

Comment:
А, что значит `comment='t'`? Можно было сделать так, тогда бы ещё 2 строки вообще не понадобилось
```suggestion
def read_gff(file_path):
    gff = pd.read_csv(file_path, sep='\t', comment='#',
                     names=['seqid', 'resourse', 'type', 'start', 'end', 'score', 'strand', 'frame', 'attribute'])
    gff['attribute'] = gff['attribute'].astype(str).apply(lambda x: x.split('_')[0].split('=')[1])
    return gff
```